src/components/Header.css

```diff
@@ -0,0 +1,27 @@
+.header-bar {
+    background-color: #3f51b5;
```

Comment:
I encourage you to use Sass pre-processor, mostly for managing variables (see [#1](https://sass-lang.com/documentation/variables), [#2](https://sass-lang.com/guide)).